tasks/main.yml

```diff
@@ -1,4 +1,13 @@
 ---
+# Note that this task is called without superuser privileges, hence it is meant to be a
+# sanity check. We don't want to create the user from this role, because we don't want to
```

Comment:
> We don't want to create the user from this role

Then this should be run in `check_mode`. See https://docs.ansible.com/ansible/latest/collections/ansible/builtin/user_module.html#attributes.

Reply:
Oh nifty, I didn't know about that! :+1: Fixed in b4fabaa.